src/test/ui/or-patterns/already-bound-name.rs

```diff
@@ -2,7 +2,7 @@
 // correctly accounts for or-patterns.
 
 #![feature(or_patterns)]
-//~^ WARN the feature `or_patterns` is incomplete
+#![allow(incomplete_features)]
```

Comment:
Fwiw, I personally prefer not to do this because it doesn't force you to remove these `#![allow(...)]`s once you remove the feature from `INCOMPLETE_FEATURES` but I'm also OK with it if my argument doesn't convince you.

Reply:
I'll probably move this to the start of the next PR and just remove or patterns from incomplete features.

Reply:
Sounds even better. :)